support-frontend/assets/pages/new-contributions-landing/components/ContributionSurvey/ContributionsSurvey.jsx

```diff
@@ -13,7 +13,7 @@ In order to display this component, a surveyLink must be
 configured below and the prop 'isRunning' needs to be set to
 `true` where the component is rendered (currently on the
 `ContributionThankYou` and `ContributionThankYouPasswordSet`)
-************************************************************/
+*********************************************************** */
```

Comment:
extra space

Reply:
eslint 👎

Reply:
(I do like eslint, to be clear - just sometimes it does unexpected things like this)